src/main/scala/mesosphere/marathon/core/task/jobs/impl/OverdueInstancesActor.scala

```diff
@@ -65,12 +65,12 @@ private[jobs] object OverdueInstancesActor {
 
       def launchedAndExpired(task: Task): Boolean = {
         task.status.condition match {
-          case Condition.Provisioned | Condition.Starting if task.status.stagedAt < unconfirmedExpire =>
+          case Condition.Provisioned | Condition.Staging if task.status.stagedAt < unconfirmedExpire =>
             logger.warn(s"Should kill: ${task.taskId} was launched " +
               s"${task.status.stagedAt.until(now).toSeconds}s ago and was not confirmed yet")
             true
 
-          case Condition.Staging if task.status.stagedAt < stagedExpire =>
+          case Condition.Starting if task.status.stagedAt < stagedExpire =>
```

Comment:
Similar here: shouldn't it be `Condition.Staging | Condition.Starting` since `taskLaunchTimeout = Time, in milliseconds, to wait for a task to enter the TASK_RUNNING state before killing it`?